<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit2"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="checkEmpty">
      <property name="text">
       <string>Проверка на пустоту</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>20</number>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="title">
        <string>Добавление</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditAdd">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>Добавить в очередь</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Удаление</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonRemove">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhSensitiveData</set>
          </property>
          <property name="text">
           <string>Удаление первого элемента</string>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonDeleteBetwMaxMin">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сделать минимальный элемент первым</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>180</height>
        </size>
       </property>
       <property name="title">
        <string>Рандомная очередь</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonNewRandomQueue">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Новая случайная очередь</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Количество:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAmount">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Минимум:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMin">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Максимум:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMax">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QListView" name="listView">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="ouptutFirstEl">
      <property name="text">
       <string>Вывод первого элемента</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
